src/js/providers/html5.ts

```diff
@@ -158,7 +158,6 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
 
         loadeddata(): void {
             checkStartDateTime();
-            VideoEvents.loadeddata.call(_this);
```

Comment:
expected removal?

Reply:
Yes. `VideoEvents.loadeddata` was a no-op, so I removed it.